xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -1776,15 +1765,18 @@ The text of a script that will be run before the build. For example:
             doc = "A dictionary of global project options.",
             mandatory = True,
         ),
+        "runner_build_file": attr.string(
+            doc = "The path to the runner's BUILD file.",
+        ),
         "runner_label": attr.string(doc = "The label of the runner target."),
         "scheme_autogeneration_mode": attr.string(
             doc = "Specifies how Xcode schemes are automatically generated.",
```

Comment:
These docs are never seen by a user. I'm not sure how much value they provide to a developer. I'm wondering if we should keep them? And if we do, should they be duplicated on the runner (and in some case triplicated if also exposed to the user in the macro) ?

Reply:
Would this prevent having the docs generated in the markdown files? If they're not supposed to be exposed to a consumer, then we could get rid of them.

Reply:
These ones aren't generated in the markdown currently. I only have them for developers.